kubectl-cloudflow/version/version.go

```diff
@@ -30,21 +30,47 @@ const ProtocolVersionKey = "protocol-version"
 // ProtocolVersionConfigMapName is the name of the configmap that contains the protocol-version
 const ProtocolVersionConfigMapName = "cloudflow-protocol-version"
 
-// FailOnProtocolVersionMismatch fails and exits if the protocol version of kubectl-cloudflow does not match with the cloudflow operator protocol version.
-func FailOnProtocolVersionMismatch() {
+// GetProtocolVersionConfigMap Get the protocol version config map set by the operator
+func GetProtocolVersionConfigMap() (*corev1.ConfigMap, error) {
 	k8sClient, k8sErr := k8s.GetClient()
 	if k8sErr != nil {
 		util.LogAndExit("Failed to create new kubernetes client, %s", k8sErr.Error())
 	}
 
 	var cm *corev1.ConfigMap
-	if configMaps, err := k8sClient.CoreV1().ConfigMaps("").List(metav1.ListOptions{}); err == nil {
+	configMaps, err := k8sClient.CoreV1().ConfigMaps("").List(metav1.ListOptions{})
```

Comment:
We could add a label to the configmap when creating it in the Cloudflow operator and then use the ListOption to select based on label.

This shows how to select on label:
https://stackoverflow.com/questions/51106923/labelselector-for-secrets-list-in-k8s

Reply:
cool .. will do .. Just curious - this implementation will work but the label will make the search efficient - right ?

Reply:
Yes, I was thinking that the cluster may have, well many configmaps so ...

Reply:
It can be fast enough with `--watch-cache` big enough from what I read https://github.com/kubernetes/community/pull/2203